### ISSUE TITLE
Slashes from .env files are parsed, newlines are not parsed properly

### ISSUE THREAD

**Issue by monim67** [nat-n/poethepoet#118]
.env file
```bash
VALUE="\\ slash works, \n newlines don't."
```

pyproject.toml
```toml
[tool.poe]
envfile = [".env"]

[tool.poe.tasks]
test = { script = "test:main" }
```

test.py
```py
import os

def main():
    print(os.getenv("VALUE"))
```

When the command is executed, newlines become "n"
```
Poe => test
\ slash works, n newlines don't.
```

**Comment by nat-n** [comment]
Hi @monim67, thanks for reporting this issue. I'll investigate.

**Comment by nat-n** [comment]
This issue is fixed (along with a few similar quirks) in [v0.18.1](https://pypi.org/project/poethepoet/0.18.1/).
